releasenotes/notes/remove-legacy-service-name-365da3952e073dd5.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    The legacy environment variable ``DD_SERVICE_NAME`` is removed.
```

Comment:
We also have tests removing `DATADOG_SERVICE_NAME`, does that need to be included here too?